specs/git.spec.js

```diff
@@ -1,3 +1,4 @@
+import "regenerator-runtime/runtime";
```

Comment:
I see this in some files, but not others. Is it needed everywhere? Is there one location it can be used and not all the others?

Reply:
I read something that they didn't add it into jest so you have to import it everywhere you use async/await tests in order to get them to run